### ISSUE TITLE
problem with thepiratebay

### ISSUE THREAD

**Issue by leo06** [cardigann/cardigann#163]
unable to download anything from thepiratebay,other trackers are ok 
installed in windows 10

**Comment by tizmagik** [comment]
Might be because of the new CloudFlare redirect?

**Comment by leo06** [comment]
sickrage is working fine with piratebay

**Comment by darkcloud784** [comment]
Is the indexer testing fine? Are you able to use cardigann to search and get results through that indexer? Dev's can't help if you don't give details, saying "I can't download from x" or "it doesn't work" isn't enough info to go on, and the only detail you gave was it was installed on windows 10, though that doesn't give what windows 10 build your on, if its pro or home, or really any specifics.

**Comment by leo06** [comment]
yes,test says ok,0 results in piratebay, did you try and is ok for you?because its not only cardigann, jackett have the same problem but sickrage is working fine and i can search and find results through my browsers

**Comment by deanochips** [comment]
i fixed it by using the new pirate bay proxy ending in vip 
(did not want to link directly, apologies for making this a bit of a treasure hunt)

**Comment by leo06** [comment]
so it was a problem caused by thepiratebay?
